stories/module-analytics-setup.stories.js

```diff
@@ -64,8 +64,8 @@ storiesOf( 'Analytics Module/Setup', module )
 
 		const setupRegistry = ( registry ) => {
 			registry.dispatch( STORE_NAME ).setSettings( {} );
-			registry.dispatch( STORE_NAME ).receiveAccounts( [] );
-			registry.dispatch( STORE_NAME ).receiveExistingTag( null );
+			registry.dispatch( STORE_NAME ).receiveGetAccounts( [] );
+			registry.dispatch( STORE_NAME ).receiveGetExistingTag( null );
 			registry.stores[ STORE_NAME ].store.dispatch( { type: 'START_FETCH_ACCOUNTS_PROPERTIES_PROFILES' } );
```

Comment:
I believe this action has changed - Storybook no longer shows a loading state here so it must have 😄

Reply:
Not sure how this passed before actually - we were passing accounts here which should be enough to not be considered loading state. So actually we shouldn't pass it here. But I fail to understand how this previously was working... 🤔

Reply:
It's this line where we manually put it in a `isDoingGetAccounts` state.
```js
registry.stores[ STORE_NAME ].store.dispatch( { type: 'START_FETCH_ACCOUNTS_PROPERTIES_PROFILES' } );
```
I fixed it 👍